docker/images/python/Dockerfile.dev

```diff
@@ -0,0 +1,14 @@
+FROM python:slim
+
+RUN apt-get update \
+    # git is needed for pip
+    && apt-get install --no-install-recommends --no-install-suggests -y \
+    git make wkhtmltopdf xvfb gettext
```

Comment:
We do not use git and make anywhere, so we should remove it.

Reply:
@artemiy312 , 

> # git is needed for pip